gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/EntityTagsService.groovy

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.netflix.spinnaker.gate.services
+
+import com.netflix.spinnaker.gate.services.commands.HystrixFactory
+import com.netflix.spinnaker.gate.services.internal.ClouddriverService
+import groovy.transform.CompileStatic
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+import org.springframework.web.bind.annotation.RequestParam
+
+@CompileStatic
+@Component
+class EntityTagsService {
+  private static final String GROUP = "entityTags"
+
+  @Autowired
+  ClouddriverService clouddriverService
+
+  List<Map> list(@RequestParam Map<String, Object> allParameters) {
+    HystrixFactory.newListCommand(GROUP, "listEntityTags") {
```

Comment:
I don't suppose it is worth partitioning the hystrix stuff by any of the parameters? (could somebody send something bunk in here that would break things for everybody)